contracts/test/Gateway.t.sol

```diff
@@ -875,14 +878,19 @@ contract GatewayTest is Test {
     function testSetPricingParameters() public {
         uint256 fee = IGateway(address(gateway)).quoteRegisterTokenFee();
         assertEq(fee, 5000000000000000);
-        // Double the exchangeRate
+        // Double both the exchangeRate and multiplier. Should lead to an 8x fee increase
         GatewayMock(address(gateway)).setPricingParametersPublic(
             abi.encode(
-                SetPricingParametersParams({exchangeRate: exchangeRate.mul(convert(2)), deliveryCost: outboundFee})
+                SetPricingParametersParams({
+                    exchangeRate: exchangeRate.mul(convert(2)),
+                    multiplier: multiplier.mul(convert(2)),
+                    deliveryCost: outboundFee
+                })
             )
         );
+        // Should expect 4x fee increase
         fee = IGateway(address(gateway)).quoteRegisterTokenFee();
-        assertEq(fee, 10000000000000000);
+        assertEq(fee, 40000000000000003);
```

Comment:
I checked the math and I am unsure where is the extra `3` comming from?

Reply:
Seems some nuance of precision loss from `_convertToNative` but does not hurt here.

Reply:
yeah, I think its do precision loss, and as Ron says its inconsequential